gril/grilreply.c

```diff
@@ -552,7 +552,7 @@ struct reply_sim_io *g_ril_reply_parse_sim_io(GRil *gril,
 	 */
 	if (message->buf_len < 12) {
 		ofono_error("Invalid SIM IO reply: size too small (< 12): %d ",
-			    (int) message->buf_len);
+				(int) message->buf_len);
```

Comment:
Please prevent whitespace changes if possible

Reply:
Fixed in 27ed62a5

Reply:
Actually fixed in 5b9a4dd5. Original file contained four spaces and not a tab